modules/appeals_api/spec/fixtures/higher_level_reviews/v0/valid_200996_headers_minimum.json

```diff
@@ -0,0 +1,8 @@
+{
+  "X-VA-SSN": "123456789",
+  "X-VA-First-Name": "Jane",
+  "X-VA-Last-Name": "Doe",
+  "X-VA-Birth-Date": "1969-12-31",
+  "X-Consumer-Username": "va.gov",
+  "X-Consumer-ID": "some-guid"
+}
```

Comment:
This file is added because before this point, we hadn't added any specs that needed it. It's an exact copy of the equivalent file in decision reviews v2